dask_cuda/tests/test_spill.py

```diff
@@ -147,3 +154,13 @@ def get_data(worker, total_size):
                 assert dc > 0
             else:
                 assert dc == 0
+
+        yield client.close()
+
+        # Ensure timeout is respected for workers
+        for w in cluster.workers:
+            yield w.close(timeout=10)
```

Comment:
Why was this necessary?  Is `yield cluster.close()` not enough?

Reply:
No, for some reason the timeout isn't respected, and I had identified where this wasn't working but now why. Now I realized that `LocalCluster` was changed significantly in distributed master, so maybe that would work now. Since we're still on 1.28.1 here, I would keep this and raise an issue to verify this with the new distributed release (1.29?), do you agree with that?

Reply:
I plan to change our testing here to point to distributed master pretty soon, so we'll have to handle that then.  Lets wait a day for that to happen and then check in here again.

Reply:
Ok, I will try doing that tomorrow then.